cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
For using the jest methods pls add `import '@testing-library/cypress/add-commands';` to this file